index.html

```diff
@@ -41,8 +41,171 @@ <h3 class="connect_me">
+                </button>
+            </div>
+
+        </section>
+
+
+        <section class="about-me">
+            <div class="about-me-intro">
+                <div class="container">
+                    <h1 class="welcome">
+                        About <br/> Myself
+                    </h1>
+                    <p class="about_me">
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.                    </p>
+                    <h3 class="connect_me">
+                        <a href="#">LET'S CONNECT</a>
+                    </h3>
+                    <ul class="icon-group">
+                        <li class="icon"> <a href="https://github.com/qusai122"><i class="fa-brands fa-github"></i></a></li>
+                        <li class="icon"> <a href="https://www.linkedin.com/in/qusai-issa-ab5a59156/"><i class="fa-brands fa-linkedin"></i></a></li>
+                        <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
+                        <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                        <li class="icon"> <a href="mailto:qusai.b.issa@gmail.com"><i class="fa-brands fa-m"></i></a></li>
+                    </ul>
+                    <button class="action-button">
+                        <h3 class="button-text">Get my resume</h3>
+                    </button>
+                </div>
+            </div>
+            <div class="my-skills">
+                <div class="container">
+                    <div class="skill">
+                        <h4 class="skill-name">Languages</h4>
+                        <i style='font-size:20px' class='fas'>&#xf107;</i>
+                    </div>
+                    <div class="sub-skills">
+                        
+                        <div class="sub-skill">
+                            <img class="sub-skill-img" src="assits/skill1.png">
+                            <h5 class="sub-skill-name">JavaScript</h5>
+                        </div>
+                        <div class="sub-skill">
+                            <img class="sub-skill-img" src="assits/skill2.png" >
+                            <h5 class="sub-skill-name">HTML</h5>
+                        </div>
+                        <div class="sub-skill">
+                            <img class="sub-skill-img" src="assits/skill3.png" >
+                            <h5 class="sub-skill-name">CSS</h5>
+                        </div>
+                    </div>
+                    <div class="skill">
+                        <h4 class="skill-name">Frameworks</h4>
+                        <i style='font-size:20px' class='fas'>&#xf105;</i>
+                    </div>
+                    <hr>
+                    <div class="skill">
```

Comment:
- [ ] Please use `ul` and `li` for skills because it is a requirement and it is also good for semantic ways. :slightly_smiling_face:
Check below :point_down: and also check more info by clicking this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)

> Please use `ul` and `li` for categories in the "About myself" section. `Languages`, `Frameworks`, and `Skills` should be listed items.